app/src/App.tsx

```diff
@@ -15,11 +15,15 @@ import BusinessGuides from "./components/UI/Pages/BusinessGuides";
 import VendorDashBoard from "./components/UI/Pages/VendorDashboard";
 import CreateVendorPage from "./components/UI/Pages/CreateVendorPage";
 import BusinessGuideArticle from "./components/UI/Molecules/BusinessGuideGUI/BusinessGuideArticle";
+import MessageError from "./components/UI/Atoms/Message/MessageError";
 
 function App(): React.ReactElement {
+  const error = useAppSelector((state) => state.root.error);
```

Comment:
`Uncaught Error: could not find react-redux context value; please ensure the component is wrapped in a <Provider>`